### PR TITLE
COWIN API SCRIPT

### PR DESCRIPTION
**Pull request by Bhuvanaa28**
## Related Issue 

- COWIN API  Script 

- [x] LGM-SOC'21 Participant 

Closes: #140 

### Describe the changes you've made

-  Added the code to obtain details of vaccination sessions
-  Added README file
- Added screenshots in Images folder

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

I executed the code for different test cases and obtained the expected output

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

img4

### CODE REVIEW

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
Changes to be done :

- Add libraries imported in "requirements.txt" file.
- Elaborate the README.md file
- Add appropriate comments in your project file wherever needed to get good understanding of it.

Kindly do the needful changes. @Bhuvanaa28

File: APIScripts/COWIN API/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# COWIN API
+
+CoWIN is a cloud-based IT solution for planning, implementation, monitoring, and evaluation of Covid-19 vaccination in India.
```
Comment (thegautamkumarjaiswal) [inline APIScripts/COWIN API/README.md:3]: correct the CoWIN to COWIN.

**Review by thegautamkumarjaiswal — CHANGES_REQUESTED** [review]
In python script, if you can add more comments will help to under code better.

File: APIScripts/COWIN API/README.md (hunk 1 above)
Comment (thegautamkumarjaiswal) [reply APIScripts/COWIN API/README.md:3]: @prathimacode-hub Please review about more comments or code description.

**Comment by Bhuvanaa28** [comment]
> Changes to be done :
> 
> * Add libraries imported in "requirements.txt" file.
> * Elaborate the README.md file
> * Add appropriate comments in your project file wherever needed to get good understanding of it.
> 
> Kindly do the needful changes. @Bhuvanaa28

I have done the necessary changes Mam

**Review by thegautamkumarjaiswal — APPROVED** [review]
All's all working! Great.

**Comment by prathimacode-hub** [comment]
@thegautamkumarjaiswal You're not allowed to review the PR's, as you being a LGMSOC participant. Kindly note this.

**Review by prathimacode-hub — APPROVED** [review]
Your PR is approved and merged successfully. You had done an amazing work. Keep rising. 🙌✔@Bhuvanaa28

**Pull request merged**